forms-web-app/jest.config.js

```diff
@@ -4,4 +4,15 @@ module.exports = {
   clearMocks: true,
   testEnvironment: 'node',
   setupFiles: ['./tests/setupTests.js'],
+  collectCoverage: true,
+  collectCoverageFrom: ['src/**/*.js'],
+  coverageReporters: ['json', 'html', 'text', 'text-summary'],
+  coverageThreshold: {
+    global: {
+      branches: 76,
```

Comment:
Presume these numbers are "where we are now"?

Reply:
indeed - and no wiggle room either, though I suspect there will need to be. Router code isn't being directly tested - maybe it should be, maybe not. Open to discussion.